gobblin-modules/gobblin-sql/src/main/java/org/apache/gobblin/writer/commands/BaseJdbcBufferedInserter.java

```diff
@@ -65,6 +67,9 @@
   protected PreparedStatement insertPstmtForFixedBatch;
   private final Retryer<Boolean> retryer;
 
+  // If this config is true, the inserter can insert duplicate primary records according to the specific language
+  protected final boolean replaceExistingValues;
```

Comment:
fair argument, I moved this field to the base class of `MySqlBufferedInserter` but not the other child classes because it is the only one that implements this logic so far. As per your comment below I have added it as a field to all the `*WriterCommands` that will fail fast if the config is turned on before creating the `*Inserter`